### PR TITLE
Update cmdlet syntax and examples according to removed external access dialing parameters

### PR DESCRIPTION
**Pull request by mskartikb**
Updating cmdlet syntax and examples to match deprecated parameters (ExternalAccessPrefix and OptimizeDeviceDialing) for external access dialing.

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTenantDialPlan.md [hunk 1]
```diff
@@ -19,34 +19,25 @@ Use the `Set-CsTenantDialPlan` cmdlet to modify an existing tenant dial plan.
...
 
 ## DESCRIPTION
-The `Set-CsTenantDialPlan` cmdlet modifies an existing tenant dial plan. A tenant dial plan determines such things as which normalization rules are applied and whether a prefix
-must be dialed for external calls. Tenant dial plans provide required information to let Enterprise Voice users make telephone calls.
+The `Set-CsTenantDialPlan` cmdlet modifies an existing tenant dial plan. A tenant dial plan determines such things as which normalization rules are applied. Tenant dial plans provide required information to let Enterprise Voice users make telephone calls.
 The Conferencing Attendant application also uses tenant dial plans for dial-in conferencing.
 
 ## EXAMPLES
 
 ### -------------------------- Example 1 --------------------------
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTenantDialPlan.md:32]:
```suggestion
### EXAMPLE 1
```

File: skype/skype-ps/skype/Set-CsTenantDialPlan.md [hunk 2]
```diff
@@ -19,34 +19,25 @@ Use the `Set-CsTenantDialPlan` cmdlet to modify an existing tenant dial plan.
...
 $nr2 = Get-CsVoiceNormalizationRule -Identity "US/US Long Distance"
-Set-CsTenantDialPlan -ExternalAccessPrefix "123" -Identity vt1tenantDialPlan9 -NormalizationRules @{Add=$nr2}
+Set-CsTenantDialPlan -Identity vt1tenantDialPlan9 -NormalizationRules @{Add=$nr2}
 ```
 
-This example updates the vt1tenantDialPlan9 tenant dial plan to have an external access prefix of 123 and use the US/US Long Distance normalization rules.
+This example updates the vt1tenantDialPlan9 tenant dial plan to use the US/US Long Distance normalization rules.
 
-### -------------------------- Example 3 --------------------------
+### -------------------------- Example 2 --------------------------
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTenantDialPlan.md:40]:
```suggestion
### EXAMPLE 2
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mskartikb, kindly commit the additional suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@mskartikb Please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
adding also @jenstrier

**Review by JensTrier — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**